README.md

```diff
@@ -91,22 +95,25 @@ Run worker from local project using Docker
 
 ```
 USAGE
-  $ aio asset-compute run-worker FILE RENDITION
+  $ aio asset-compute run-worker [FILE] [RENDITION] [-v] [--version] [-a <value>] [-d <value> | -P <value> | -p
```

Comment:
Could we keep the order defined originally in Nui? 
`aio asset-compute run-worker [FILE] -P <value> [RENDITION]`? So our internal tools stay compatible/can switch easily.

Reply:
this is auto-generated from code, so it always takes the form `<command> [arguments...] [flags...]`
In use, I believe the flags can be before the args or mixed